src/components/Home.js

```diff
@@ -0,0 +1,62 @@
+const Home = () => (
```

Comment:
- [ ] Kindly consider adding A <Books> component, which contains the following components:


 `<BookList>`

- Renders a list of books using the <Book> component



`<Book>`

- Takes the book data as a prop and renders the title and the author

- Has a button for removing books

💀 the button wouldn't remove the books yet, this is just a placeholder for now

doing so will help you to meet the below requirements